### PR TITLE
Add Comments to Local Benchmark and Update Favicon

### PR DESCRIPTION
**Pull request by wrighkv1**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7381)

### CODE REVIEW

**Review by pyu10055 — APPROVED** [review]
Thank you Kristen!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7381#-:-NOLEYL95E21NVFjs9GE:b-olwcrz)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @chunnienc)

File: e2e/benchmarks/local-benchmark/loader.js [hunk 1]
```diff
@@ -29,10 +42,20 @@ function loadScript(url) {
   })
 }
 
+/**
+ * processUrls is called from loadTFJS(localBuild)
+ * processUrls is used to replace the urls in the urls array with local urls 
+ * if the localBuild array includes the package name
+ * 
+ * @param {string array} localBuild
+ * @param {string array} urls
```
Comment (chunnienc) [inline e2e/benchmarks/local-benchmark/loader.js:51]: JSDoc has a syntax for array types: https://jsdoc.app/tags-type.html

**Review by chunnienc — CHANGES_REQUESTED** [review]
Nice, thanks!

File: e2e/benchmarks/local-benchmark/loader.js (hunk 1 above)
Comment (wrighkv1) [reply e2e/benchmarks/local-benchmark/loader.js]: Thanks. Fixed JSDoc syntax for array types.

**Review by chunnienc — APPROVED** [review]
LGTM. Thanks!

**Pull request closed without merging**